# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/sandbox/chromium/sandbox/win/src/policy_broker.cc
# Commit: 370c40e49230
# Full Hash: 370c40e4923073a87845b57c1f4851bacd9663da
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2019-06-12 21:42:35
# Regressor Bug: 1552160
# File Overlap Count: 3
# Description:
#   Bug 1552160 Part 1: Roll-up of chromium sandbox update and mozilla patches to get a running browser. r=jld,aklotz,tjr,bobowen
#   
#   This updates security/sandbox/chromium/ files to chromium commit 84108231f6e6e0772fb9a4643679ce76aa771e67.
#   
#   Existing and new patches applied from security/sandbox/chromium-shim/patches/with_update/ to give a compiling and mostly working browser.
# ==============================================================================

diff -r bd5ecd72f76c -r 370c40e49230 security/sandbox/chromium/sandbox/win/src/policy_broker.cc
--- a/security/sandbox/chromium/sandbox/win/src/policy_broker.cc	Wed Jun 12 21:55:59 2019 +1200
+++ b/security/sandbox/chromium/sandbox/win/src/policy_broker.cc	Wed Jun 12 11:10:48 2019 +0100
@@ -2,12 +2,12 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+#include "sandbox/win/src/policy_broker.h"
+
 #include <stddef.h>
 
 #include <map>
 
-#include "sandbox/win/src/policy_broker.h"
-
 #include "base/logging.h"
 #include "base/win/pe_image.h"
 #include "base/win/windows_version.h"
@@ -28,17 +28,17 @@
 // This is the list of all imported symbols from ntdll.dll.
 SANDBOX_INTERCEPT NtExports g_nt;
 
-#define INIT_GLOBAL_NT(member) \
+#define INIT_GLOBAL_NT(member)                          \
   g_nt.member = reinterpret_cast<Nt##member##Function>( \
-                      ntdll_image.GetProcAddress("Nt" #member)); \
-  if (NULL == g_nt.member) \
-    return false
+      ntdll_image.GetProcAddress("Nt" #member));        \
+  if (!g_nt.member)                                     \
+  return false
 
-#define INIT_GLOBAL_RTL(member) \
-  g_nt.member = reinterpret_cast<member##Function>( \
-                      ntdll_image.GetProcAddress(#member)); \
-  if (NULL == g_nt.member) \
-    return false
+#define INIT_GLOBAL_RTL(member)                                                \
+  g_nt.member =                                                                \
+      reinterpret_cast<member##Function>(ntdll_image.GetProcAddress(#member)); \
+  if (!g_nt.member)                                                            \
+  return false
 
 bool InitGlobalNt() {
   HMODULE ntdll = ::GetModuleHandle(kNtdllName);
@@ -71,14 +71,14 @@
   return true;
 }
 
-bool SetupNtdllImports(TargetProcess *child) {
+bool SetupNtdllImports(TargetProcess* child) {
   if (!InitGlobalNt()) {
     return false;
   }
 
 #ifndef NDEBUG
   // Verify that the structure is fully initialized.
-  for (size_t i = 0; i < sizeof(g_nt)/sizeof(void*); i++)
+  for (size_t i = 0; i < sizeof(g_nt) / sizeof(void*); i++)
     DCHECK(reinterpret_cast<char**>(&g_nt)[i]);
 #endif
   return (SBOX_ALL_OK == child->TransferVariable("g_nt", &g_nt, sizeof(g_nt)));
@@ -106,8 +106,7 @@
                     20))
     return false;
 
-  if (!INTERCEPT_NT(manager, NtOpenThreadTokenEx, OPEN_THREAD_TOKEN_EX_ID,
-                    24))
+  if (!INTERCEPT_NT(manager, NtOpenThreadTokenEx, OPEN_THREAD_TOKEN_EX_ID, 24))
     return false;
 
   if (!is_csrss_connected) {